src/Apps/Order/Components/__tests__/SavedAddresses.jest.tsx

```diff
@@ -204,9 +170,7 @@ describe("Saved Addresses", () => {
       expect(modal.props().show).toBe(false)
       button.simulate("click")
 
-      setTimeout(() => {
-        expect(modal).toHaveLength(1)
-      }, 0)
+      expect(modal).toHaveLength(1)
```

Comment:
I really don't know how this kind of misusage assertion inside a timeout passed through review, we should be aware that doesn't work and make the test assertion a false positive 🚓